dispatch/static/manager/src/js/components/PollEditor/Poll.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from 'react'
+import { Button, Intent } from '@blueprintjs/core'
+
+require('../../../styles/components/poll.scss')
+
+const COLOR_OPACITY = .8
+
+class Poll extends Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      showResults: true,
+    }
+  }
+
+  componentDidMount() {
+    this.update()
+  }
+
+  update() {
+    let answers = []
+    let votes = []
+    let pollQuestion = this.props.question ? this.props.question : 'Poll Question'
+    if(this.props.answers){
+      for(let answer of this.props.answers){
```

Comment:
Add a space after the `for` -- this is just a stylistic thing that differentiates it from a functional. (It should probably be a rule in the linter)

Reply:
i think https://eslint.org/docs/rules/space-after-keywords addresses this